src/libs/actions/Session/index.js

```diff
@@ -348,7 +348,7 @@ function setPassword(password, validateCode, accountID) {
             }
 
             // This request can fail if the password is not complex enough
-            Onyx.merge(ONYXKEYS.ACCOUNT, {error: response.message});
+            Onyx.merge(ONYXKEYS.ACCOUNT, {errors: {[DateUtils.getMicroseconds]: response.message}});
```

Comment:
needs to be `DateUtils.getMicroseconds()` with the parens.

For all of these - do we need the square brackets around `DateUtils.getMicroseconds()` - it's not an array, is it? 

Lastly, I think this is me not fully grokking Onyx.merge yet, but won't this completely overwrite any existing `errors`? Since we're merging we don't overwrite the rest of the onyx key, but won't it overwrite any already existing object at `errors`?

Reply:
Good catch on parens!

Regarding squares bracket, I think it is just bracket notation of ECMA script for dynamic keys 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors#bracket_notation

Regarding Onyx merge, it won't overwrite errors and only append them. More info here https://github.com/Expensify/react-native-onyx#merging-data
You can also try in dev by putting two Onyx merge error statements one after the another. Make sure to increase random seconds in the second statement so they have different keys.